pkg/mimirpb/timeseries.go

```diff
@@ -704,9 +704,10 @@ func (p *WriteRequest) ForIndexes(indexes []int, initialMetadataIndex int) *Writ
 	}
 
 	return &WriteRequest{
-		Timeseries: timeseries,
-		Metadata:   metadata,
-		Source:     p.Source,
+		Timeseries:              timeseries,
+		Metadata:                metadata,
+		Source:                  p.Source,
+		SkipLabelNameValidation: p.SkipLabelNameValidation,
```

Comment:
Not to reviewers: not a bug today, because `SkipLabelNameValidation` is only read in distributors before calling `ForIndexes` but I think it's better to fix it.